src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughMultipleReliefPatterns.java

```diff
@@ -0,0 +1,26 @@
+/*
+FallThrough
+checkLastCaseGroup = true
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughMultipleReliefPatterns {
+
+  void method(int i) {
+    while (true) {
+      switch (i) {
+        case 5: {
+          i++;
+        }
+        /* block */ /* fallthru */ // comment
+        case 6:
+          i++;
+          /* block */ /* fallthru */ // comment
+
```

Comment:
Looks like this for 

>Can you point to keywords where the suppression text is not the last comment on the line. If we don't have one, then create it and point to it.
Example:
/* trigger */ /* comment */ // last
/* comment */ /* fallthru */ // last

Reply:
@rnveach , think this address your point https://github.com/checkstyle/checkstyle/pull/14734#discussion_r1597719839

If not , please restate your point on new PR.

Reply:
My comment was on documentation, which this comment is not pointing at. It looks documentation is different so I will need to re-review.